package com.springapicalls.RestApiCalls.repository;

import com.springapicalls.RestApiCalls.model.Greeting;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface GreetingRepository extends JpaRepository<Greeting, Long> {
    // You can define custom query methods here if needed
}